src/test/java/jalau/at18/katas/Team2/Test2048/BoardTest.java

```diff
@@ -0,0 +1,172 @@
+package jalau.at18.katas.Team2.Test2048;
```

Comment:
Test files are under incorrect folder hierarchy, please fix so they are under
```
src/test/java/jalau/at18/katas/game2048/team2
```

Reply:
we modified the line in the file with the correct path